server/reaction_hooks.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+// ReactionHasBeenAdded is invoked after the reaction has been committed to the database.
+//
+// Note that this method will be called for reactions added by plugins, including the plugin that
+// added the reaction.
+//
+// This demo implementation logs a message to the demo channel whenever a reaction is added to a post.
+func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	configuration := p.getConfiguration()
+
+	if configuration.disabled {
+		return
+	}
+
+	user, err := p.API.GetUser(reaction.UserId)
+	if err != nil {
+		p.API.LogError("failed to query user", "user_id", reaction.UserId)
+		return
+	}
+
+	post, err := p.API.GetPost(reaction.PostId)
+	if err != nil {
+		p.API.LogError("failed to query post", "post_id", reaction.PostId)
```

Comment:
Nit pick: Usually log messages are capitalized 
```suggestion
		p.API.LogError("Failed to query post", "post_id", reaction.PostId)
```

Reply:
@hanzei - its lowercase throughout this plugin repo. I can go ahead and make it capitalized in the other files too if you're OK w/ that being bundled w/ this PR (so it all remains consistent).

Reply:
Good point. I didn't know that. Consistent is key here. If you want, you can change all instances, if not it can stay as it is.

Reply:
Went ahead and changed all `LogError` and `LogWarn` instances that I found, so everything's consistent.